EngineLayer/FdrAnalysis/FdrAnalysisEngine.cs

```diff
@@ -12,7 +13,8 @@ public class FdrAnalysisEngine : MetaMorpheusEngine
         private readonly double ScoreCutoff;
         private readonly string AnalysisType;
 
-        public FdrAnalysisEngine(List<PeptideSpectralMatch> psms, int massDiffAcceptorNumNotches, CommonParameters commonParameters, List<string> nestedIds, string analysisType = "PSM") : base(commonParameters, nestedIds)
+        public FdrAnalysisEngine(List<PeptideSpectralMatch> psms, int massDiffAcceptorNumNotches, CommonParameters commonParameters,
```

Comment:
Do we need common Parameters as well as the list of file specific common parameters?

Reply:
for now yes. I think that it may be possible to eliminate common params. But, file specific is an incomplete set and we'd have to deal w/ some things being different between the different files.